src/assets/components/Tile.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function Tile({title, text, image}){
+    return(
+        <section>
+            {title && <h2>{title}</h2>}
```

Comment:
Hier heb je het zo te zien iets anders aangepakt met die && operator tov van de uitwerkingen maar dat maakt natuurlijk niet uit. Ik moet voor mezelf nog eens gaan kijken hoe dat precies werkt want of ik ben het vergeten of we hebben dat niet in deze bootcamp gehad:)